api/idea-card/documentation/1.0.0/idea-card.json

```diff
@@ -664,6 +665,10 @@
                 },
                 "description": {
                   "type": "string"
+                },
+                "isHidden": {
```

Comment:
So is this field supposed to hide the open roles? If so, wouldn't it be better to name it isOpenRolesHidden for clarity?